doc/source/tutorials.rst

```diff
@@ -30,9 +29,6 @@ Intermediate
 :doc:`Design Patterns Tutorial <tutorials/designpatterns>`
     Discusses some general aspects of kOS flow control and optimizations.
 
-:doc:`PID Loop Tutorial <tutorials/pidloops>`
-    Starts with a basic proportional feedback loop and develops, in stages, a complete PID-loop to control the throttle of a simple rocket design.
-
```

Comment:
Disagree.  Are you saying the old pid loop tutorial should be completely removed?  It's not broken or wrong.  It still works, and teaches how a PID works.  It just uses a homebrewed PID instead of the built-in one.  One could argue that a new version needs to be written that uses the built-in PID structure, but that doesn't mean that it's wrong to explain what a PID loop is doing under the hood by making a homemade one from scratch.

Reply:
My intent here was to remove the old tutorial completely.  I definitely think a new version which uses the built-in structure would be valuable, but I think there are other explanations of PID loops that can be referenced here, such as the Wikipedia entry https://en.wikipedia.org/wiki/PID_controller linked in the subreddit's sidebar.

Reply:
I'm okay with this, but I figure the same PR that deletes the old explanation should also be the PR that adds the new alternative for it.  Can you put this link reference in there to replace the old page?  (I'm not a fan of "break something in this PR, and we'll repair it later in a second PR" because of how often one PR gets merged and the other one doesn't.)

Reply:
Ideally I'd like to write a new PID page that isn't as hard to read as most pages written by mathematicians, like the wiki page, are, but that can come later as long as the wiki link is put in here for now.  The wiki link isn't any worse than the current page is, so go ahead and remove it and replace it with the wiki link.